bindings/rust/bench/README.md

```diff
@@ -19,13 +19,17 @@ certs/generate_certs.sh
 cargo bench --config aws-lc-config/s2n.toml
 ```
 
+### Features
+
+Default features (`rustls` and `openssl`) can be disabled by running the benches with `--no-default-features`. The non-default `memory` and `historical-perf` features are used to enable dependencies specific to those types of benches, and are automatically used by the scripts that run those benches.
```

Comment:
Prob dont need this section. Features are pretty well defined in the cargo docs so I dont think we need to explain it here.

Reply:
I think this would be good for someone that usually uses C but wants to do benching, but I don't have strong opinions on it.